src/flash/flashCmd.ts

```diff
@@ -79,5 +84,14 @@ export async function verifyCanFlash(
       new Error("NOT_SELECTED_BAUD_RATE")
     );
   }
+  const selectedFlashType = idfConf.readParameter("idf.flashType");
+  if (selectedFlashType === "DFU") {
+    if (!(await listDfuDevices())) {
```

Comment:
What kind of information you get from `dfu-util --list` ? Is it possible to validate that the device defined for current target (esp32, esp32s2) is listed in `listDfuDevices()` output ?

Reply:
This is an example of what I receive from `dfu-util --list`  I receive for esp32s2:
`Found Runtime: [303a:0002] ver=0723, devnum=1, cfg=1, intf=2, path="0-1", alt=0, name="UNKNOWN", serial="0"`

I don't think I can validate it